docs/user/alerting/pre-configured-connectors.asciidoc

```diff
@@ -14,11 +13,16 @@ action are predefined, including the connector name and ID.
 - Appear in all spaces because they are not saved objects.
 - Cannot be edited or deleted.
 
+Sensitive configuration information, such as credentials, can use the <<creating-keystore, {kib} keystore>>.
+
+A preconfigured action types has only preconfigured connectors. But preconfigured connectors could belong to the preconfigured action type or to the regular action type as well.
+
 [float]
 [[preconfigured-connector-example]]
-=== Example of a preconfigured connector
+=== Creating a preconfigured connector
 
-The following example shows a valid configuration 2 out-of-the box connector.
+The following example shows a valid configuration of two out-of-the box connectors: <<slack-action-type, Slack>> and <<webhook-action-type, Webhook>>.
+You can find all details about available action types configuration <<action-types, here>>.
```

Comment:
I don't think this sentence "You can find all details..." is necessary and suggest removing it.